packages/paste-website/src/pages/primitives/modal-dialog-primitive/index.mdx

```diff
@@ -44,7 +44,7 @@ export const pageQuery = graphql`
   name="Modal Dialog Primitive"
   categoryRoute={SidebarCategoryRoutes.PRIMITIVES}
   githubUrl="https://github.com/twilio-labs/paste/tree/main/packages/paste-core/primitives/modal-dialog"
-  storybookUrl="https://twilio-labs.github.io/paste/?path=/story/primitives-modaldialog--custom-overlay-and-content"
+  storybookUrl="https://main--5e53448165911c0022e68c74.chromatic.com/?path=/story/primitives-modaldialog--custom-overlay-and-content"
```

Comment:
nit: maybe we can make this url a constant and hardcode this as storybookBaseUrl in the ComponentHeader and only pass in the `/?path=...` to make this easier to update and reference?

Reply:
This is a good idea, Imma experiment